README.md

```diff
@@ -1,5 +1,10 @@
 ## Eppo Multiplatform: SDKs and Artifacts to support Flagging and Experimentation
 
+[![Test](https://github.com/Eppo-exp/eppo-multiplatform/actions/workflows/ci.yml/badge.svg)](https://github.com/Eppo-exp/eppo-multiplatform/actions/workflows/ci.yml)  
```

Comment:
Thanks for doing the work to split them out!

Made the change to the text here, but from what I can gather, it's actually pulling the name from the yaml file. I've updated the name in the yaml file, but since the badge is pulling the status from `main`, it will still say "Test" until we merge here.